lib/args.ts

```diff
@@ -18,7 +18,6 @@ export function parse(args: string[], ...extractors: Extractor[]): string[] & {
 			args = extractor(params, args);
 			if (arg0 !== args[0]) break;
 		}
-		if (args0 === args[0]) break;
```

Comment:
This seems wrong. The purpose of this line is:

Assume the following call:

```
const args = parse([ '--valid', '123', '--invalid' , '456'], string('valid'));
// args => [ '--invalid', '456' ] with `params` = { valid: '123' }
```

Without this line the result would be

`[ '345' ]` with `params: { valid: '123' }`

So I guess I don't quite understand what it is you actually want to fix. Can you explain further?